packages/netlify/src/ssr-function.ts

```diff
@@ -36,14 +36,16 @@ export const createExports = (manifest: SSRManifest, _args: Args) => {
 			}
 
 			if (integrationConfig.cacheOnDemandPages) {
+				const isCacheableMethod = ['GET', 'HEAD'].includes(request.method);
```

Comment:
Are `GET`, `HEAD` the only cacheable methods?

Reply:
They're not the only cacheable ones, but the ones that most caching proxies cache by default. Caching a POST request can be a footgun because they tend to have side effects, same goes for PUT or DELETE requests. GET and HEAD are the only methods where caching is mostly safe.